From: Ian Campbell <ijc@debian.org>
Date: Fri, 21 Nov 2014 13:45:51 +0000
Subject: allow libxl to find default path to pygrub.

The Xen debian packages relocate pygrub into
/usr/lib/xen-X.Y/bin/pygrub, not /usr/bin/pygrub. Since libxl knows to
DTRT with a bare "pygrub" just use that by default.

--- a/src/libxl/libxl_conf.h
+++ b/src/libxl/libxl_conf.h
@@ -53,7 +53,7 @@
 # define LIBXL_LIB_DIR LOCALSTATEDIR "/lib/libvirt/libxl"
 # define LIBXL_SAVE_DIR LIBXL_LIB_DIR "/save"
 # define LIBXL_DUMP_DIR LIBXL_LIB_DIR "/dump"
-# define LIBXL_BOOTLOADER_PATH BINDIR "/pygrub"
+# define LIBXL_BOOTLOADER_PATH "pygrub"
 
 /* libxl interface for setting VCPU affinity changed in 4.5. In fact, a new
  * parameter has been added, representative of 'VCPU soft affinity'. If one
